bin/client/main.rs

```diff
@@ -433,6 +433,10 @@ async fn run() -> std::result::Result<(), ClientError> {
                 }
             }
         }
+        SubCommand::Clean(_clean_opts) => {
+            // TODO: Clean up cached bindles
```

Comment:
You missing a commit here?

Reply:
Oh, just saw you move it to draft. Never mind

Reply:
@thomastaylor312 sorry Im working on it, this is still in draft